components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/workflow/ApplicationDeletionSimpleWorkflowExecutor.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
License format used is not up-to-date. Please fix. Fix all other similar places as well.